nao_localization/ObjectMsg object
---
bool success
